core/serialisation/src/main/java/uk/gov/gchq/gaffer/serialisation/implementation/raw/RawDoubleSerialiser.java

```diff
@@ -39,27 +39,34 @@ public boolean canHandle(final Class clazz) {
     public byte[] serialise(final Double d) throws SerialisationException {
         final byte[] out = new byte[8];
         final long value = Double.doubleToRawLongBits(d);
-        out[0] = (byte) ((int) (value & 255));
-        out[1] = (byte) ((int) (value >> 8) & 255);
-        out[2] = (byte) ((int) (value >> 16) & 255);
-        out[3] = (byte) ((int) (value >> 24) & 255);
-        out[4] = (byte) ((int) (value >> 32) & 255);
-        out[5] = (byte) ((int) (value >> 40) & 255);
-        out[6] = (byte) ((int) (value >> 48) & 255);
-        out[7] = (byte) ((int) (value >> 56) & 255);
+        out[0] = (byte) ((value & 255));
```

Comment:
I'm not sure about these casting changes in these files. Can we leave it as it was?